### PR TITLE
Issue 6435 - move more tests to FULL mode

### PR DESCRIPTION
**Pull request by tevans78**
#build

### CODE REVIEW

File: dev/com.ibm.ws.microprofile.config_repeat_tests/src/com/ibm/ws/microprofile/config/fat/repeat/RepeatConfigDefaults.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.config_repeat_tests/src/com/ibm/ws/microprofile/config/fat/repeat/RepeatConfigDefaults.java]: 2019!

**Comment by tevans78** [comment]
#build

File: dev/fattest.simplicity/src/componenttest/custom/junit/runner/TestModeFilter.java [hunk 2]
```diff
@@ -73,6 +73,17 @@ public boolean shouldRun(Description desc) {
...
+
+        if (!run) {
+            Log.debug(getClass(), "Removing test " + desc.getMethodName() + " with mode " + testMode + " from list to run, because not valid for current mode "
+                                  + FRAMEWORK_TEST_MODE);
+            return false;
+        } else
+            return true;
+    }
+
+    public static boolean shouldRun(TestMode testMode) {
```
Comment (Azquelt) [inline dev/fattest.simplicity/src/componenttest/custom/junit/runner/TestModeFilter.java]: Not sure that this change is actually an improvement

Comment (tevans78) [reply dev/fattest.simplicity/src/componenttest/custom/junit/runner/TestModeFilter.java]: it isn't except that i can call the new method from my code

**Comment by tevans78** [comment]
Failures in PB are all completely unrelated to this FAT change. The Config FAT tests appear to be running correctly in the FULL and LITE modes.

**Pull request merged**